GCRCatalogs/catalog_configs/dc2_object_run2.1.1i_matched.yaml

```diff
@@ -1,5 +1,7 @@
 subclass_name: dc2_matched_table.DC2MatchedTable
-filename: '/global/projecta/projectdirs/lsst/groups/SSim/DC2/matched_ids_run2.1.1i_dpdd.fits.gz'
```

Comment:
I propose that we use this pull request to fix the NERSC-specific paths problem. @yymao @boutigny @heather999 @wmwv